sdk/monitor/query/azmetrics/README.md

```diff
@@ -33,12 +34,14 @@ The clients default to the Azure public cloud. For other cloud configurations, s
 
 #### Create a client
 
-Example client
+Example [client][example_client]
 
 ## Key concepts
 
 [Azure Monitor Metrics][metrics_overview] collects numeric data from monitored resources into a time series database. Metrics are numerical values that are collected at regular intervals and describe some aspect of a system at a particular time. Metrics are lightweight and capable of supporting near real-time scenarios, making them particularly useful for alerting and fast detection of issues.
 
+Note, each resource must be in the same region as the endpoint passed in when instantiating the client, and each resource must be in the same Azure subscription. Furthermore, the [metric namespace][supported_metrics] that contains the metrics to be queried must also be passed.
```

Comment:
I'd omit making these points here because they're more API documentation than high-level concepts and I see they're covered in the constructor and QueryResources docs where they're pertinent and actionable

Reply:
removed in [07629df](https://github.com/Azure/azure-sdk-for-go/pull/22769/commits/07629df511a25816b6525ca77dfb30991141ab86)